src/pint/models/timing_model.py

```diff
@@ -498,6 +498,16 @@ def __getattr__(self, name):
             f"Attribute {name} not found in TimingModel or any Component"
         )
 
+    def __setattr__(self, name, value):
+        """Mostly this just sets ``self.name = value``.  But in the case where they are both :class:`Parameter` instances
+        with different names, this copies the ``quantity``, ``uncertainty``, ``frozen`` attributes only.
+        """
+        if isinstance(value, Parameter) and name != value.name:
+            for p in ["quantity", "uncertainty", "frozen"]:
+                setattr(getattr(self, name), p, getattr(value, p))
+        else:
+            self.__dict__[name] = value
+
```

Comment:
I think this should not work for incompatible parameters.

For example, `m.F0 = m.F1` should throw an error. But `m.JUMP1=m.JUMP2` should work.
Maybe we can check the units?

Reply:
I think the incompatible units will be rejected by the quantity setting.  So I don't think an explicit unit check is required.  But I put a test in to demonstrate this.